app/services/transcription_service.py

```
@@ -238,6 +253,25 @@ def _try_gpt4o_transcribe(self, video_url: str, language: str = None) -> Tuple[O
                 "success": False
             }
     
+    def _log_cost_usage(self, video_id: str, service_type: str, cost_amount: float, 
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)